custom/icds/messaging/custom_recipients.py

```diff
@@ -51,6 +51,22 @@ def recipient_mother_person_case_from_ccs_record_case(case_schedule_instance):
         return None
 
 
+def recipient_mother_person_case_from_ccs_record_case_excl_migrated_or_opted_out(case_schedule_instance):
+    from custom.icds.messaging.custom_content import person_case_is_migrated_or_opted_out
+    try:
+        mother = mother_person_case_from_ccs_record_case(case_schedule_instance.case)
+    except CaseRelationshipError as e:
+        if not skip_notifying_missing_ccs_record_parent(e):
+            notify_exception(None, message="ICDS ccs_record relationship error")
+
+        return None
```

Comment:
i was thinking we'd just call `recipient_mother_person_case_from_ccs_record_case` and use that result as the starting point in this function

Reply:
Of course, DRY. https://github.com/dimagi/commcare-hq/pull/21504/commits/3663bdd3903ae373ae6fefe85ddf5a3bf95696cc